utils/test/TestKickStart.bat

```diff
@@ -1,11 +1,46 @@
+@ECHO OFF
+rem ------------------------------------------------------------------------------
+rem  Copyright 2015 Esri
+rem  Licensed under the Apache License, Version 2.0 (the "License");
+rem  you may not use this file except in compliance with the License.
+rem  You may obtain a copy of the License at
+rem 
+rem    http://www.apache.org/licenses/LICENSE-2.0
+rem 
+rem  Unless required by applicable law or agreed to in writing, software
+rem  distributed under the License is distributed on an "AS IS" BASIS,
+rem  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+rem  See the License for the specific language governing permissions and
+rem  limitations under the License.
+rem ------------------------------------------------------------------------------
+rem  TestKickStart.bat
+rem ------------------------------------------------------------------------------
+rem  requirments:
+rem  * ArcGIS Desktop 10.X+ or ArcGIS Pro 1.X+
+rem  * Python 2.7 or Python 3.4
+rem  author: ArcGIS Solutions
+rem  company: Esri
+rem ==================================================
+rem  description:
+rem  This file starts the test running for Desktop (Python 2.7+) and
+rem  ArcGIS Pro (Python 3.4+).
+rem 
+rem ==================================================
+rem  history:
+rem  10/06/2015 - MF - placeholder
+rem  10/30/2015 - MF - tests running
+rem ==================================================
 
-@ECHO ON
+REM =====================================================
+REM If you have BOTH versions of Python installed:
 
-REM ECHO Python 2.7 Tests ===============================
-REM py -2.7 TestRunner.py
+ECHO Python 3.4 Tests ===============================
+py -3.4 TestRunner.py
 
-REM ECHO Python 3.4 Tests ===============================
-REM py -3.4 TestRunner.py
+ECHO Python 2.7 Tests ===============================
+py -2.7 TestRunner.py
```

Comment:
The only issue I see here is I don't think the Windows Python launcher (py.exe) is installed unless you have Python 3.3+ installed.